app/services/model_controller/notifier.rb

```diff
@@ -24,6 +25,18 @@ def success_update(item)
       end
     end
 
+    def success_list_update(item, field_name)
+      respond_to do |format|
+        format.html { redirect_to admin_edit_uri(item) }
+        format.json do
+          field = RademadeAdmin::Model::Graph.instance.model_info(item.class).data_items.data_item(field_name)
+          success_message(item, 'Поле обновлено!', {
```

Comment:
Redundant curly braces around a hash parameter.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.